src/map/clif.c

```diff
@@ -17527,6 +17536,26 @@ void clif_party_leaderchanged(struct map_session_data *sd, int prev_leader_aid,
 }
 
 /**
+ * Decrypt packet identifier for player
+ * @param fd
+ * @param sd
+ * @param packet_ver
+ * Orig author [Ind/Hercules]
+ **/
+static unsigned short clif_parse_cmd(int fd, struct map_session_data *sd) {
+#ifndef PACKET_OBFUSCATION
+	return RFIFOW(fd, 0);
+#else
```

Comment:
Mind to add some comments here why this things are done?
I will add some suggestions